lib/dor/datastreams/embargo_metadata_ds.rb

```diff
@@ -36,7 +36,7 @@ def self.xml_template
     def to_solr(solr_doc = {}, *args)
       solr_doc = super
       rd20 = twenty_pct_release_date
-      ::Solrizer.insert_field(solr_doc, 'embargo_release', release_date.first.utc.strftime('%FT%TZ'), :dateable) unless release_date.blank?
+      ::Solrizer.insert_field(solr_doc, 'embargo_release', release_date.first.utc.strftime('%FT%TZ'), :dateable) unless release_date&.first.blank?
```

Comment:
I don't think this is correct.  if `release_date` returns `[nil]` then this isn't going to prevent the error.  Can you write a test?

Reply:
```
Dor::Item.new.embargoMetadata.release_date
=> [nil]
```

Reply:
Oh, actually it would prevent the error, but the `&.` is unnecessary.

Reply:
release_date returns `[nil]` and I started to write a test but ... isn't this just for the other code with empty data?  would we ever an empty release date?

Reply:
🤷‍♂ Reviewer didn't ask me to.

Reply:
I'm suggesting `[nil].first.blank?`

Reply:
yes, that is what is broken and what this fixes.  I can make a test but I'm asking if that is realistic in a fedora embargo datastream ...

Reply:
I'm saying get rid of the `&` as that doesn't do anything

Reply:
@jcoyne and fixed again again.